exec/java-exec/src/main/java/org/apache/drill/exec/metastore/analyze/MetadataAggregateContext.java

```diff
@@ -63,8 +67,8 @@ public boolean createNewAggregations() {
   }
 
   @JsonProperty
-  public List<SchemaPath> excludedColumns() {
-    return excludedColumns;
+  public List<SchemaPath> nonSchemaColumns() {
```

Comment:
Maybe "metadataColumn" or "implicitColumns"? From a pure implementation perspective, every column is a schema column for some schema. Please pick a name that explains their use: "statisticsColumns"?

And, thank you for renaming this. I would have guessed that "excludedColumns" were those in the file that are not being projected in this query, which would then have seemed very odd.